explore/src/main/scala/explore/targets/TargetColumns.scala

```diff
@@ -149,7 +149,8 @@ object TargetColumns:
             t => targetBrightnesses.get(t).flatMap(_.get(band))
           ).setCell(_.value.map(displayWithoutError(_)(displayBrightness)).orEmpty)
             .setSize(80.toPx)
-            .setEnableSorting(false) // We cannot sort since there may be different units.
+            // By user request we allow sorting by value though there maybe a mix of units
+            .sortableBy(_.map(_.value))
```

Comment:
I think Andy expected sorting to group rows with the same unit and then sort by value within them.

Maybe we can define an `Ordering[Measure[N]]` given an `Ordering[N]` and `Ordering[Units]` ?

Reply:
I discussed this and it seems just the name is enough as by their range they'd naturally group by units.
@andrewwstephens WDYT?

Reply:
I am hopeful that simply sorting by value will be good enough.